src/Popover/index.js

```diff
@@ -0,0 +1,75 @@
+// @flow
+import * as React from "react";
+import styled from "styled-components";
+
+import defaultTokens from "../defaultTokens";
+import Portal from "../Portal";
+import ClickOutside from "../ClickOutside";
+import PopoverContentWrapper from "./helpers/ContentWrapper";
+import type { Props, State } from "./index.flow";
+
+const StyledPopoverChild = styled.div`
+  position: relative;
+`;
+StyledPopoverChild.defaultProps = {
+  theme: defaultTokens,
+};
+
+class Popover extends React.PureComponent<Props, State> {
+  state = {
+    shown: false,
+  };
+
+  handleIn = () => {
+    this.setState({ shown: true });
+  };
+
+  handleOut = () => {
+    this.setState({ shown: false });
+  };
+
+  handleClick = () => {
+    this.setState({ shown: !this.state.shown });
+  };
+
+  handleClickOutside = () => {
+    this.timeoutOutside = setTimeout(() => {
+      this.setState({ shown: false });
+    });
+  };
+
+  handleClickContent = () => {
+    clearTimeout(this.timeoutOutside);
+  };
+
+  container: { current: any | HTMLDivElement } = React.createRef();
+  timeoutOutside: TimeoutID;
+
+  render() {
+    const { shown } = this.state;
+    const { children, content, preferredPosition, prefferedAnchorPosition } = this.props;
+    const { handleClickContent, handleOut, container } = this;
+
+    return (
+      <React.Fragment>
+        <StyledPopoverChild onClick={this.handleClick} ref={container}>
+          <ClickOutside onClickOutside={this.handleClickOutside}>{children}</ClickOutside>
+        </StyledPopoverChild>
+        {shown && (
+          <Portal>
```

Comment:
We should use `element="popovers"` as for other components - modals, tooltips. We can work with that much more easier when comes to z-indexes across whole product.